server/django_server/settings.py

```diff
@@ -41,6 +41,8 @@
     'DEFAULT_PERMISSIONS_CLASSES': (
         'rest_framework.permission.IsAuthenticated'
     ),
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
+    'PAGE_SIZE': 25,
```

Comment:
Having line 44, will mess with retrieving from all the other APIs, so when I log in as a sys admin, retrieving organizations and manage members, returns no organizations or employees respectively. 
Can you set these parameter explicitly in the view so that we don't have these adverse side effects?

Reply:
I added a local class for the pagination of the items such that it doesn't affect the rest of the paginations